### PR TITLE
Adding Mobile menu using JavaScript DOM manipulation

### PR DESCRIPTION
**Pull request by zmekonnen251**
## Description

In this PR, we built a mobile menu for [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) using **JavaScript** , **CSS** and **HTML**.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS & JavaScript requirements

- [x] Followed list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] Followed list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### Project requirements

- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [x] Used pair-programming.

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const anchorTag = document.createElement('a');
+    anchorTag.innerText = mobileMenuList[i];
+    anchorTag.href = `#${mobileMenuList[i].toLowerCase()}`;
+    anchorTag.className = 'mobile_menu_links';
+    element.appendChild(anchorTag);
+    ul.appendChild(element);
+  }
+  nav.appendChild(xIcon);
+  nav.appendChild(ul);
+  header.appendChild(nav);
```
Comment (moise10r) [inline script.js]: - [ ] Great job so far :clap: however the mobile menu should overfly the home section and not be displayed before the home :point_down: design  [template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@zmekonnen251,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️  
- Good commit messages ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zmekonnen251** [comment]
- We have fixed the overfly of the mobile menu

**Review by rustam-tolipov — APPROVED** [review]
## Hi TEAM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**